pkg/daemon/ceph/client/image.go

```diff
@@ -41,8 +41,15 @@ type CephBlockImage struct {
 	InfoName string `json:"name"`
 }
 
-func ListImages(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]CephBlockImage, error) {
+func ListImagesInPool(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]CephBlockImage, error) {
+	return ListImagesInRadosNamespace(context, clusterInfo, poolName, "")
+}
+
+func ListImagesInRadosNamespace(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, namespace string) ([]CephBlockImage, error) {
 	args := []string{"ls", "-l", poolName}
```

Comment:
not related to this PR we could have used `rbd ls -p=replicapool` instead of `-l`

Reply:
Yep. Just used the the existing method we had for list images.